conference-demos/onnxruntime-openvino-lcm-demo/README.md

```diff
@@ -0,0 +1,52 @@
+# Image generation using Latent Consistency Model and ONNX Runtime with OpenVINO Execution Provider 
+
+LCMs: The next generation of generative models after Latent Diffusion Models (LDMs). 
```

Comment:
Check the content for grammar issues and typos e.g. Grammarly

Reply:
fixed